.github/workflows/publish-release.yaml

```diff
@@ -45,9 +49,20 @@ jobs:
       - name: Download release artifacts
         run: |
           mkdir -p artifacts
-          gh release download "${{ github.ref_name }}" --dir ./artifacts --pattern 'esc-*'
+          gh release download "${ESC_VERSION}" --dir ./artifacts --pattern 'esc-*'
           find artifacts
       - name: Publish Blobs
         run: |
           aws s3 sync artifacts s3://get.pulumi.com/esc/releases --acl public-read
 
+  dispatch:
+    name: ${{ matrix.job.name }}
+    if: inputs.run-dispatch-commands && !contains(inputs.version, '-')
+    runs-on: ubuntu-latest
+    needs: [publish]
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+          - name: Dispatch docs workflow
+            run-command: pulumictl create cli-docs-build "${ESC_VERSION}" --event-type "esc-cli"
```

Comment:
will this command be okay with the format of `ESC_VERSION` as-is? I have a hard time keeping track of where `v1.2.3` is okay and where things want `1.2.3`.